kafka/kafka-manager.sh

```diff
@@ -13,159 +13,96 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
-# This script installs Kafka Manager (https://github.com/yahoo/kafka-manager)
+# This script installs CMAK formerly only as Kafka Manager
 # on a Dataproc cluster.
 #
 # Kafka Manager server will be running on the first master node (port 9000 by default).
 
 set -euxo pipefail
 
-readonly ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
-readonly KAFKA_MANAGER_VERSION="$(/usr/share/google/get_metadata_value attributes/kafka-manager-version || echo 2.0.0.2)"
 readonly KAFKA_MANAGER_HTTP_PORT="$(/usr/share/google/get_metadata_value attributes/kafka-manager-http-port || echo 9000)"
-readonly KAFKA_VERSION="$(/usr/share/google/get_metadata_value attributes/kafka-version || echo 1.1.1)"
-readonly KAFKA_ENABLE_JMX="$(/usr/share/google/get_metadata_value attributes/kafka-enable-jmx || echo false)"
-
-readonly KAFKA_MANAGER_HOME="/opt/kafka-manager-${KAFKA_MANAGER_VERSION}"
-readonly KAFKA_MANAGER_CONFIG="${KAFKA_MANAGER_HOME}/conf/application.conf"
-readonly KAFKA_MANAGER_GIT_URI=https://github.com/yahoo/kafka-manager.git
+readonly KAFKA_MANAGER_GIT_URI=https://github.com/yahoo/CMAK.git
 readonly KAFKA_MANAGER_GIT_DIR=/tmp/kafka-manager
-readonly ZOOKEEPER_HOME=/usr/lib/zookeeper
+readonly KAFKA_MANAGER_VERSION="cmak-3.0.0.7"
+readonly KAFKA_MANAGER_HOME="/opt/kafka-manager-${KAFKA_MANAGER_VERSION}"
 readonly ZOOKEEPER_CONFIG=/etc/zookeeper/conf/zoo.cfg
 
-function download_kafka_manager() {
-  git clone --branch "${KAFKA_MANAGER_VERSION}" --depth 1 "${KAFKA_MANAGER_GIT_URI}" "${KAFKA_MANAGER_GIT_DIR}"
+function install_packages(){
+   apt-get update
+   apt-get install -yq apt-transport-https curl gnupg
 }
 
-function build_kafka_manager() {
-  pushd ${KAFKA_MANAGER_GIT_DIR}
-  ./sbt clean dist
-  mv target/universal/kafka-manager-${KAFKA_MANAGER_VERSION}.zip /opt/
-  rm -rf "${KAFKA_MANAGER_GIT_DIR}"
-  popd
+function add_sources(){
+   echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | sudo tee /etc/apt/sources.list.d/sbt.list
```

Comment:
Init actions run as root, so `sudo` is not necessary. (fix other places as well).